File: app/src/main/java/com/tifone/tnews/api/IMobileNewsApi.java
package com.tifone.tnews.api;

import com.tifone.tnews.bean.news.MultiNewsArticleBean;

import io.reactivex.Observable;
import okhttp3.ResponseBody;
import retrofit2.Call;
import retrofit2.Response;
import retrofit2.http.GET;
import retrofit2.http.Query;

/**
 * Created by tongkao.chen on 2018/5/7.
 */

public interface IMobileNewsApi {

    @GET("http://is.snssdk.com/api/news/feed/v62/?iid=5034850950&device_id=6096495334&refer=1&count=20&aid=13")
    Observable<MultiNewsArticleBean> getNewsArticle(
            @Query("category") String category,
            @Query("max_behot_time") String max_behot_time);

    @GET("http://lf.snssdk.com/api/news/feed/v62/?iid=12507202490&device_id=37487219424&refer=1&count=20&aid=13")
    Observable<MultiNewsArticleBean> getNewsArticle2(
            @Query("category") String category,
            @Query("max_behot_time") String max_behot_time);

    @GET("http://lf.snssdk.com/api/news/feed/v62/?iid=12507202490&device_id=37487219424&refer=1&count=20&aid=13")
    Call<ResponseBody> getArticle3(@Query("category") String category,
                                   @Query("max_behot_time") String max_behot_time);
}

File: app/src/main/kotlin/com.tifone.tnews/api/IMobileNewsApi.kt
package com.tifone.tnews.api

import com.tifone.tnews.bean.news.MultiNewsArticleBean
import com.tifone.tnews.bean.news.MultiNewsArticleDataBean
import io.reactivex.Observable
import retrofit2.http.GET
import retrofit2.http.Query

interface IMobileNewsApi {

    @GET("http://is.snssdk.com/api/news/feed/v62/?iid=5034850950&device_id=6096495334&refer=1&count=20&aid=13")
    fun getNewsArticle(
            @Query("category") category: String?,
            @Query("max_behot_time") max_behot_time: String
    ): Observable<MultiNewsArticleBean>

    @GET("http://lf.snssdk.com/api/news/feed/v62/?iid=12507202490&device_id=37487219424&refer=1&count=20&aid=13")
    fun getNewsArticle2(
            @Query("category") category: String?,
            @Query("max_behot_time") max_behot_time: String
    ): Observable<MultiNewsArticleBean>
}